### ISSUE TITLE
Trying to install via SCCM

### ISSUE THREAD

**Issue by will-admin** [glpi-project/glpi-agent#128]
### Documentation

Yes, I read it

### Solution search

Yes, I read it

### Professional support

Yes, I know

### Is your help request related to a problem?

When I try to deploy the agent via a bat script within SCCM. I just get a 1619 error which states it cannot open the application. But when I launch the same bat file interactively on a random machine. It installs on that machine with no errors. here is the script with the bat file.

Running: msiexec /a
"\\<server name>\Applications\GLPI\GLPI-Agent-1.2-x64.msi" /quiet ADD_FIREWALL_EXCEPTION=1 RUNNOW=1 SERVER='http://<Server name>/front/inventory.php' ADD_FIREWALL_EXCEPTION=1

Please don't respond with "you need to actually enter a server name" :D 
![SCCM error](https://user-images.githubusercontent.com/106243073/170248018-0cb75d9a-3cd5-4b94-b10d-23e3034666fc.png)

### Expected behavior

Agent to deploy to all machines

### Actions you've considered

Tried many different switches with the script but as I said. This bat runs and installs the agent with no issues when done interactively.

**Comment by g-bougard** [comment]
Hi @will-admin 

try to use `/i` in place of `/a` as first msiexec option.

**Comment by will-admin** [comment]
> Hi @will-admin
> 
> try to use `/i` in place of `/a` as first msiexec option.

**Comment by will-admin** [comment]
Tried that also. it doesn't do /i if installing from a network share even interactively

**Comment by g-bougard** [comment]
Is it possible to first copy the MSI locally and then make a local install ?

**Comment by g-bougard** [comment]
Did you check what means the error code ?

Took from https://docs.microsoft.com/en-us/windows/win32/msi/error-codes
```
ERROR_INSTALL_PACKAGE_OPEN_FAILED | 1619 | This installation package could not be opened. Verify that the  package exists and is accessible, or contact the application vendor to  verify that this is a valid Windows Installer package.
```

Make sure the SCCM client has the right to open the MSI file from that share.

**Comment by will-admin** [comment]
Yes mate. I have even applied “EVERYONE” to the particular root folder with full control. And making sure I am running SCCM with a DA account.

**Comment by will-admin** [comment]
This may be my only choice but I was hoping not too as in the future I want to look at possibly using the nightly backups agents for future updates.

**Comment by g-bougard** [comment]
Please, don't answer by email, this make your answer hard to follow as it includes a complex signature weirdly rendered by GH... or just avoid the signature when answering by email.

I don't really understand your "hope". Anyway your issue seems really to be a SCCM or MSI installation issue. I hope you'll find your way to make it work.

**Comment by g-bougard** [comment]
Closing without feedback.
Feel free to reopen or just comment.
